packages/core/src/graphql/warehouses-orders/warehouses-orders.types.graphql

```diff
@@ -8,6 +8,7 @@ input OrderCreateInput {
 	warehouseId: String!
 	products: [OrderProductCreateInput!]!
 	options: WarehouseOrdersRouterCreateOptions
+	orderType: Int
```

Comment:
@YavorGrancharov hm, I think we should be using some sort of enum or something like that... Int is not a good type for sure....